support/Sources/GraphQL.swift

```diff
@@ -294,6 +294,22 @@ public struct SchemaViolationError: Error {
 	}
 }
 
+public enum InputValue<T> {
+	// Serialzeable value
+    case defined(T?)
+	// Not serializable
+    case undefined
+    
+    @available(*, deprecated, message: "Use `?? .undefined` instead")
+    public init(orUndefined optional: Optional<T>)  {
```

Comment:
InputValue didn't exist before, so why add a deprecated initializer?

Reply:
That probably shouldn't be there.